test/controllers/FileChecksControllerSpec.scala

```diff
@@ -96,8 +96,30 @@ class FileChecksControllerSpec extends FrontEndTestHelper {
         frontEndInfoConfiguration
       )
       val recordsPage = controller.recordProcessingPage(consignmentId).apply(FakeRequest(GET, s"/consignment/$consignmentId/records"))
-      playStatus(recordsPage) mustBe SEE_OTHER
-      redirectLocation(recordsPage).get must startWith(s"/consignment/$consignmentId/records-results")
+      playStatus(recordsPage) mustBe OK
+      contentAsString(recordsPage) must include("data-module=\"govuk-notification-banner\" >")
+      contentAsString(recordsPage) must not include("govuk-button--disabled")
+    }
+
+    "render the notification banner and enable the button if the file checks are complete and all checks are not successful" in {
```

Comment:
Is that correct, if the checks are not successful the button is enabled?

Reply:
Yeah. If they fail, you still get the notification saying your checks are done. You click continue and then you get the failure page.